README.md

```diff
@@ -11,14 +11,8 @@ Add `luhn` under `[dependencies]` in your `Cargo.toml`:
 luhn = "0.1.0"
 ```
 
-Require the library:
-
-```rust
-extern crate luhn;
```

Comment:
Is this omitted because it's obvious enough how to require it?

Reply:
In rust 2018 `extern crate` became optional. (They made this change because listing the same dependencies in both Cargo.toml and the root source file is kinda redundant.)  It's still valid to include these statements - 2018 edition is backward-compatible with 2015 edition - but it's no longer necessary.

Reply:
Nice thx! TIL